src/System.CommandLine.Tests/TestApps/NativeAOT/NativeAOT.csproj

```diff
@@ -1,21 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
-	
-	<PropertyGroup>
-		<OutputType>Exe</OutputType>
-		<TargetFramework>$(TargetFrameworkForNETSDK)</TargetFramework>
-		<!-- producing more detailed log output -->
-		<TrimmerSingleWarn>false</TrimmerSingleWarn>
-    <PublishAot>true</PublishAot>
-	</PropertyGroup>
 
-	<PropertyGroup>
-		<SystemCommandLineDllPath Condition="'$(SystemCommandLineDllPath)'==''">..\..\..\System.CommandLine\bin\Release\$(TargetFrameworkForNETSDK)\System.CommandLine.dll</SystemCommandLineDllPath>
-	</PropertyGroup>
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>$(TargetFrameworkForNETSDK)</TargetFramework>
+        <!-- producing more detailed log output -->
+        <TrimmerSingleWarn>false</TrimmerSingleWarn>
+        <PublishAot>true</PublishAot>
+        <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
@agocke @MichalStrehovsky - is there a reason why this isn't enabled by default with `PublishAot`?

Reply:
> @agocke @MichalStrehovsky - is there a reason why this isn't enabled by default with `PublishAot`?

We follow platform conventions - Control Flow Guard requires an explicit opt-in with VC++ too.